.yarnrc.yml

```diff
@@ -15,3 +15,5 @@ plugins:
     spec: '@yarnpkg/plugin-exec'
 
 yarnPath: .yarn/releases/yarn-3.0.1.cjs
+
+npmPublishRegistry: https://registry.npmjs.org/
```

Comment:
probably not necessary because it is the default config

Reply:
no, yarn uses the yarn registry by default

Reply:
yes, and it is an alias to npm, see https://yarnpkg.com/getting-started/qa#why-registryyarnpkgcom-does-facebook-track-us